examples/app_drive_mounted_s3/app.py

```diff
@@ -0,0 +1,165 @@
+"""Demonstrates how a Drive can be used to mount an AWS s3 bucket to a Work's filesystem.
```

Comment:
Could you move this example to the tests directly? There are too many asserts to be considered as an example.
Or would you mind merging this one with the other drive example?